pages/api/webhooks/order-created.ts

```diff
@@ -1,21 +1,171 @@
 import type { Handler } from "retes";
 
 import { Response } from "retes/response";
-import { toNextHandler } from "retes/adapter";
+// import { toNextHandler } from "retes/adapter";
 import { withSaleorEventMatch } from "@saleor/app-sdk/middleware";
 
-import { withSaleorDomainMatch } from "../../../lib/middlewares";
+import {
+  withSaleorDomainMatch,
+  withWebhookSignatureVerified,
+} from "../../../lib/middlewares";
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
@zaiste that's the most horrible part of this proof of concept, I directly copied here multiple functions from `retes` and `next`, ultimately we could hide that in `retes`, but need to have a conversation how to do that.

Topic for discussions:
 * how to expose `config: { api: { bodyParser: false }}}` from inside `retes`? is it possible (it's used here: https://github.com/vercel/next.js/blob/40cf3ce63a18ceb3f0551377b24d0bf1bcb916f4/packages/next/server/api-utils/node.ts#L181)
 * can we replace `apiResolver` anyhow in `next`? would that be maintainable and will not cause confusion? (I'm talking about that: https://github.com/vercel/next.js/blob/40cf3ce63a18ceb3f0551377b24d0bf1bcb916f4/packages/next/server/api-utils/node.ts#L161)